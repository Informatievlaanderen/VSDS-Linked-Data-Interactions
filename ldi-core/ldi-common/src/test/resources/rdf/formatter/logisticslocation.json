{
  "@context": [
    "https://vocabulary.uncefact.org/unece-context.jsonld",
    "https://geojson.org/geojson-ld/geojson-context.jsonld",
    {
      "isVersionOf": "http://purl.org/dc/terms/isVersionOf",
      "generatedAtTime": {
        "@id": "http://www.w3.org/ns/prov#generatedAtTime",
        "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
      }
    }
  ],
  "type": "LogisticsLocation",
  "isVersionOf": "unece:identifier/node/1",
  "generatedAtTime": "2020-05-20T10:00:00Z",
  "id": "unece:identifier/node/1/1",
  "name": "k1700",
  "statusCode": "0",
  "geometry": {
    "type": "Point",
    "coordinates": [
      4.9041,
      52.3676
    ]
  },
  "servicingSpecifiedParty": {
    "type": "LocationParty",
    "id": "unece:identifier/servicingSpecifiedParty/1",
    "name": "DP-World"
  }
}