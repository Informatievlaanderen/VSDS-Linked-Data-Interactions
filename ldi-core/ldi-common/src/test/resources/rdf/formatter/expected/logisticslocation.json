{
  "@context": "https://geojson.org/geojson-ld/geojson-context.jsonld",
  "id": "https://vocabulary.uncefact.org/identifier/node/1/1",
  "type": "https://vocabulary.uncefact.org/LogisticsLocation",
  "http://purl.org/dc/terms/isVersionOf": "unece:identifier/node/1",
  "http://www.w3.org/ns/prov#generatedAtTime": {
    "type": "http://www.w3.org/2001/XMLSchema#dateTime",
    "@value": "2020-05-20T10:00:00Z"
  },
  "geometry": {
    "type": "Point",
    "coordinates": [
      4.9041,
      52.3676
    ]
  },
  "https://vocabulary.uncefact.org/name": {
    "type": "http://www.w3.org/2001/XMLSchema#string",
    "@value": "k1700"
  },
  "https://vocabulary.uncefact.org/servicingSpecifiedParty": {
    "id": "https://vocabulary.uncefact.org/identifier/servicingSpecifiedParty/1",
    "type": "https://vocabulary.uncefact.org/LocationParty",
    "https://vocabulary.uncefact.org/name": {
      "type": "http://www.w3.org/2001/XMLSchema#string",
      "@value": "DP-World"
    }
  },
  "https://vocabulary.uncefact.org/statusCode": {
    "type": "http://www.w3.org/2001/XMLSchema#string",
    "@value": "0"
  }
}